README.md

```diff
@@ -126,7 +126,7 @@ To execute transactions using a *CPK* instance, call `execTransactions` with an
 
 Each of the `transactions` provided as input to this function must be an *Object* with the following properties:
 
-* `operation`: Either `CPK.CALL` (0) or `CPK.DELEGATECALL` (1) to execute the transaction as either a normal call or a delegatecall. Note: when connected to Gnosis Safe via WalletConnect, this property is ignored, and `CPK.CALL` is assumed. Optional property, `CPK.CALL` is the default value.
+* `operation`: Either `CPK.CALL` (0) or `CPK.DELEGATE_CALL` (1) to execute the transaction as either a normal call or a delegatecall. Note: when connected to Gnosis Safe via WalletConnect, this property is ignored, and `CPK.CALL` is assumed. Optional property, `CPK.CALL` is the default value.
```

Comment:
If we're changing DELEGATECALL to DELEGATE_CALL, then we might as well go all the way and use [TS's naming conventions](https://www.typescriptlang.org/docs/handbook/enums.html) (Call, DelegateCall).